includes/class-wc-coupon.php

```diff
@@ -828,25 +718,15 @@ public function read_manual_coupon( $code, $coupon ) {
 		$this->set_props( $coupon );
 	}
 
-	/*
-    |--------------------------------------------------------------------------
-    | Other Actions
-    |--------------------------------------------------------------------------
-    */
-
 	/**
 	 * Increase usage count for current coupon.
 	 *
 	 * @param string $used_by Either user ID or billing email
 	 */
 	public function inc_usage_count( $used_by = '' ) {
-		if ( $this->get_id() ) {
-			$this->data['usage_count']++;
-			update_post_meta( $this->get_id(), 'usage_count', $this->get_usage_count() );
-			if ( $used_by ) {
-				add_post_meta( $this->get_id(), '_used_by', strtolower( $used_by ) );
-				$this->set_used_by( (array) get_post_meta( $this->get_id(), '_used_by' ) );
-			}
+		if ( $this->get_id() && $this->data_store ) {
+			$this->set_prop( 'usage_count', ( $this->get_usage_count( 'edit' ) + 1 ) );
+			$this->data_store->inc_usage_count( $this, $used_by );
```

Comment:
Since this method is moving to a data store, can we name it better? inc_ is hard to understand.